tox.ini

```diff
@@ -149,13 +151,13 @@ deps =
     # restrictions in their deps, so we need to pin old markupsafe here too.
     markupsafe < 2.1.0
     robotexclusionrulesparser
-    Pillow
+    Pillow; python_version < '3.13'
     Twisted[http2]
     uvloop; platform_system != "Windows"
     bpython  # optional for shell wrapper tests
     brotli; implementation_name != 'pypy'  # optional for HTTP compress downloader middleware tests
     brotlicffi; implementation_name == 'pypy'  # optional for HTTP compress downloader middleware tests
-    zstandard; implementation_name != 'pypy'  # optional for HTTP compress downloader middleware tests
+    zstandard; implementation_name != 'pypy' and python_version < '3.13'  # optional for HTTP compress downloader middleware tests
```

Comment:
To remove after a full cffi release is made (planned in July) and zstandard wheels are built with it.